packages-web/datagrid/src/components/Pagination.tsx

```diff
@@ -1,13 +1,15 @@
 import { createElement, ReactElement } from "react";
 
 export interface PaginationProps {
+    canNextPage: boolean;
+    canPreviousPage: boolean;
     gotoPage: (page: number) => void;
-    previousPage: () => void;
     nextPage: () => void;
-    page: number;
+    numberOfItems?: number;
```

Comment:
you have `numberOfItems` and `numberOfPages`. Shouldn't be one of the be sufficient? Especially since you are passing a page size as well

Reply:
Fixed